packages/user-operation-controller/src/types.ts

```diff
@@ -315,10 +315,10 @@ export type SmartContractAccount = {
  */
 export type UserOperationReceipt = {
   /** Confirmed total cost of the gas for the user operation. */
-  actualGasCost: string;
+  actualGasCost: string | number;
```

Comment:
I wonder if we should use the `Hex` type here instead of `string` for clarity?

Reply:
Done in https://github.com/MetaMask/core/pull/4161/commits/1e28a31da243f52098c5ce4536e905848a3f04b1